### ISSUE TITLE
Accessibility - Poor Color Contrast of Lime Green and White Text

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Closure-App#1]
The color contrast of the lime green background ( #7ED321 ) and white text ( #ffffff ) is 1.87:1 and needs to be 4.5:1 or higher to pass.

If you want to use this green please make the text #3d3d3d which gives you a color contrast of 5.81:1. Making this change applies to any other screen that has this color combination.

Please review screenshot below. The extra red text is my notes on the image to help people out.

![san deigo status page - poor color contrast with lime green background and white text](https://user-images.githubusercontent.com/10423487/34781920-203eefb4-f5f5-11e7-92c0-f15fbc8a8788.png)

**Comment by georgepliu** [comment]
Text color is now changed to black for better contrast.

This issue has been resolved and is reflected in the current build #11

**Comment by jfc3-dol** [comment]
With black text on the lime green ( #7ED320 ) background the color contrast ratio is now 11.23:1 and needed to be 4.5:1 or higher to pass.

I'm closing this issue since it's resolved.
